### PR TITLE
Show metric samples

### PR DESCRIPTION
**Pull request by OAGr**
image

### CODE REVIEW

File: src/components/metrics/modal/index.js [hunk 1]
```diff
@@ -17,6 +18,18 @@ const percentages = (values, perc) => {
   return perc.map(e => { return {percentage: e, value: percentile(values, values.length, e)} })
 }
 
+const SampleList = ({samples}) => (
+  <ul className='SampleList'>
+    {_.map(samples, (element, index) => {
+      return (
+        <li key={index}>
+          <div key={index}>{element}</div>
```
Comment (mmcdermott) [inline src/components/metrics/modal/index.js]: Mostly just for my own curiosity, why does the `div` need a `key` prop?

File: src/components/metrics/modal/index.js [hunk 2]
```diff
@@ -109,6 +123,21 @@ export class MetricModal extends Component {
...
               </div>
+              <div className='col-sm-2'>
+
+              <div className='metricModal--sample-container'>
+                <DropDown
+                  headerText={'Samples'}
+                  openLink={<a className='modal-action button'> Samples </a>}
+                  position='right'
+                  hasPadding={true}
+                  ref='DropDown'
```
Comment (mmcdermott) [inline src/components/metrics/modal/index.js]: Tiny thing, but can we not add a `ref` here? Sometimes `ref`s cause problems and this one doesn't appear used, is all.

File: src/components/metrics/modal/index.js (hunk 1 above)
Comment (OAGr) [inline src/components/metrics/modal/index.js]: I believe React generally does for lists, I copied this from the data field where this was required.

File: src/components/metrics/modal/index.js (hunk 2 above)
Comment (OAGr) [inline src/components/metrics/modal/index.js]: Will do.

**Pull request merged**